agents/product-name-researcher/product-name-researcher.yml

```
@@ -0,0 +1,161 @@
+# Product Name Research Agent Configuration
+# Specializes in comprehensive product name validation including domain availability,
+# trademark searching, social media handle checking, and competitive analysis
+
+mcpServers:
+  # Domain availability checking
+  domain-checker:
+    type: stdio
+    command: uvx
+    args:
+      - truffle-ai-domain-checker-mcp
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)